apps/src/code-studio/initApp/clientApi.js

```diff
@@ -121,10 +121,20 @@ var base = {
    *        collection object.
    */
   fetch: function(childPath, callback) {
+    this.fetchWithDataType(childPath, callback, 'json');
+  },
+
+  /**
+   * Retrieve a collection.
+   * @param {string} childPath The path underneath api_base_url
+   * @param {AjaxNodeStyleCallback} callback - Expected result is the requested
+   *        collection object.
+   */
+  fetchWithDataType: function(childPath, callback, dataType) {
     $.ajax({
       url: this.api_base_url + '/' + childPath,
       type: 'get',
-      dataType: 'json'
+      dataType: dataType
     })
```

Comment:
nit: could keep this as one function, but just have the `dataType` param default to `'json'` if not specified? (e.g. `fetch(childPath, callback, dataType = 'json')`)

Reply:
done!